setup/ansible/rhel8.yml

```diff
@@ -0,0 +1,161 @@
+# rhel8.yml
+# Ansible Playbook to deploy OBS according to 
+# https://github.com/OSEC-pl/obs/blob/master/setup/rhel8.txt
+#
+# Copyright 2020 Steven Bonneville 
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Configure OBS on RHEL 8
+  hosts: all
+  become: yes
+
+  vars:
+    rhel8_build_tools:
+      - git
+      - vim
+      - tmux
+      - "@Development Tools"
+      - kernel-devel
+      - kernel-headers
+      - elfutils-libelf-devel
+      - cmake
+      - qt5-qtbase-devel
+
+  tasks:
+    - name: Kernel is up to date
+      yum:
+        name: kernel
+        state: latest
+      register: kernel_updated
+
+    - name: Reboot if kernel updated
+      reboot:
+      when: kernel_updated['changed'] == true
+
+    - name: All packages are up to date
+      yum:
+        name: '*'
+        state: latest
+
+    # Could use another condition if it's been done
+    - name: Make sure PowerTools is enabled on CentOS 
+      command: 
+        cmd: dnf config-manager --enable PowerTools
+      when: ansible_distribution == 'CentOS'
+
+    - name: Build tools are present
+      yum:
+        name: "{{ rhel8_build_tools }}"
+        state: latest
+
+    - name: Make sure EPEL is enabled
+      yum:
+        name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+        state: latest
+        disable_gpg_check: yes
+
+    - name: Make sure RPMFusion is enabled
+      yum:
+        name:
+          - https://download1.rpmfusion.org/free/el/rpmfusion-free-release-8.noarch.rpm
+          - https://download1.rpmfusion.org/nonfree/el/rpmfusion-nonfree-release-8.noarch.rpm
+        state: latest
+        disable_gpg_check: yes
+
+    - name: Make sure OBS is installed
+      yum:
+        name:
+          - obs-studio
+          - obs-studio-devel
+        state: latest
+
+    - name: Confirm /srv/buildtree directory exists
+      file:
+        path: /srv/buildtree
+        state: directory
+        mode: 1777
```

Comment:
Isn't that too open? 1755 will be better choice IMO

Reply:
I agree. Also it would be better to use a directory in $HOME to avoid any security implications. A possible candidate would be `$HOME/.cache`.
Ideally, the packages would be built in Copr or somewhere else so that they could just be installed here.

Reply:
Yeah, I started with it tighter and in /srv instead of /tmp because I didn't want to figure out where to do it in /home and I didn't want to make too many assumptions about system layout.  It wouldn't be too hard to change this up, but I figured I'd throw it together from the checklist and we could tune it from there.